utils/pom.xml

```diff
@@ -38,6 +38,42 @@
             <artifactId>config</artifactId>
             <version>${typesafe.config.version}</version>
         </dependency>
+        <dependency>
```

Comment:
`spark-core`, `spark-sql`, `org.apache.hadoop` are all duplicated from the master `pom.xml` file. Is that intentional?

Reply:
Good point, this was a left-over from trying to make it work on HDP sandbox